fastchat/model/model_adapter.py

```diff
@@ -654,7 +654,7 @@ class OasstPythiaAdapter(BaseModelAdapter):
     """The model adapter for OpenAssistant/oasst-sft-4-pythia-12b-epoch-3.5"""
 
     def match(self, model_path: str):
-        return "oasst" in model_path and "pythia" in model_path
+        return "oasst" in model_path.lower() and "pythia" in model_path.lower()
```

Comment:
```suggestion
        model_path = model_path.lower()
        return "oasst" in model_path and "pythia" in model_path
```
If you use `model_path.lower()` multiple times, extract it as a common expression.
Apply this style to your all changes.